pulser-core/pulser/backend/config.py

```diff
@@ -256,6 +260,16 @@ def is_time_in_evaluation_times(
             np.any(np.abs(np.array(evaluation_times, dtype=float) - t) <= tol)
         )
 
+    def _to_abstract_repr(self) -> dict[str, Any]:
+        return self._backend_options
+
+    def to_abstract_repr(self, validate: bool = False) -> str:
```

Comment:
Normally we validate by default, I think that is preferable. Also, to mimic the signature of other cases, it would be better to use
```suggestion
    def to_abstract_repr(self, skip_validation: bool = False) -> str:
```

Reply:
Done